### ISSUE TITLE
Single Product FSE Template > Add to cart with option block > button styling is missing / blockification not complete

### ISSUE THREAD

**Issue by Marc-pi** [woothemes/woocommerce#42211]
## Describe the bug

The Single Product Template encapsulate a special area called Add to Cart with option, that encapsulate a Add to cart button.
* The button's style can not be altered, it's too compact since encapculated.
We cannot alter it from Editor style (customize the blocks) neither
* The button's color in the Site Editor (template editing) does not match the true color visible on front (end user side) : standard WP Blue vs your site button color palette

## To reproduce

1. Edit the Single Product Template
2. Click on the Add to Cart with Option block within the View List

![image](https://github.com/woocommerce/woocommerce-blocks/assets/919332/6e0e73c8-dbb7-4a37-a2d2-164a9b879b41)

## Expected behavior

* When we click on this block from the Product Catalog Template for instance, to reach the Product Collection block, we have the styling options for the button

![image](https://github.com/woocommerce/woocommerce-blocks/assets/919332/17ecd20b-4da9-4c0e-b16e-0140a14c3453)

* When we click from single product, we do not have them > we should > seems the blockification is not complete there

 
![image](https://github.com/woocommerce/woocommerce-blocks/assets/919332/7092b5e2-b6d4-4840-babc-2051850243a9)

* In site Style Editor, this special block is not reachable to alter its style > seems it is an early draft block, waiting for complete blockification

![image](https://github.com/woocommerce/woocommerce-blocks/assets/919332/25a40d6a-6211-4dcb-a151-fdec9139dbbb)

## Environment

- WP 6.3.1
- Woo 8.1.1
- Woo Blocks 11.1.0
- Gutenberg 16.6.0
- TT3 theme

**Comment by sunyatasattva** [comment]
Hello @Marc-pi !

Thanks for the thorough examination. These are indeed known limitations of the block. We wanted to push out a blockified version of the template as fast as we could, but we are still using pieces of the PHP template under the hood.

Your analysis, however, is deeply appreciated and I'll add it to our backlog (I'll replace the `bug` label, with the `enhancement` label, however).

**Comment by Marc-pi** [comment]
@sunyatasattva hello Lucio
you're welcome
My need about this `Add to Cart with options` massive block is to be able to : 
- alter the way the information is styled (this issue)
- be able to improve the display of child elements in case of **grouped products** 
    - Add `SKU` per child line (under the line as additional information)
    - Add an `Add to cart` button per line (i doubt the old hooks still work), and its look (-/+ buttons with default quantity to 1)
    - Be able to add custom information within the child lines (additional to the child element product name), taken from the child product data

see https://github.com/woocommerce/woocommerce/issues/42279

**Comment by sunyatasattva** [comment]
@Marc-pi Since the block uses classic PHP templates under the hood, you may be able to achieve your goals by using legacy hooks and filters for the time being to modify the block content.

**Comment by Marc-pi** [comment]
@sunyatasattva yep, thanks for this confirmation, that's the plan for today's work ;-)

**Comment by gigitux** [comment]
@Marc-pi I'm going to close this issue given that we are working to blockify the Add to Cart block. This is the tracking issue: https://github.com/woocommerce/woocommerce/issues/53014

cc @Aljullu @mdperez86
